include/swift/ClangImporter/CXXMethodBridging.h

```diff
@@ -62,16 +62,20 @@ struct CXXMethodBridging {
   }
 
   NameKind classifyNameKind() {
-    bool allLower = llvm::all_of(getClangName(), islower);
+    bool hasUpper = std::any_of(getClangName().begin(), getClangName().end(), std::isupper);
```

Comment:
still getting that same error:
```
home/build-user/swift/include/swift/ClangImporter/CXXMethodBridging.h:65:21: error: no matching function for call to 'any_of'
    bool hasUpper = std::any_of(getClangName().begin(), getClangName().end(), std::isupper);
```

Reply:
What if you put std::isupper inside a closure, like this:
```
llvm::all_of(getClangName(), [](unsigned char ch) { return std::isupper(ch); })
```